tests/parser/test_ctffind.py

```diff
@@ -79,3 +80,14 @@ def test_invalid_job_references_raise_KeyErrors(ctffind):
 
 def test_astigmatism(ctffind):
     assert ctffind["job003"][0].astigmatism == "288.135742"
+
+
+@pytest.mark.skipif(sys.platform == "win32", reason="does not run on windows")
+def test_diagnosis_plot_path(proj, ctffind):
+    assert ctffind["job003"][0].diagnostic_plot_path == str(
+        pathlib.PurePosixPath(proj.basepath)
```

Comment:
I believe `.basepath` is guaranteed to be a path object, so you shouldn't need to recast it?

Reply:
This is a hangover from when I was trying to get the test to work on Windows, although I ultimately gave up. I'll remove it